smallrye-reactive-messaging-rabbitmq/src/main/java/io/smallrye/reactive/messaging/rabbitmq/RabbitMQConnector.java

```diff
@@ -119,6 +119,9 @@
 @ConnectorAttribute(name = "dead-letter-queue-type", direction = INCOMING, description = "If automatically declare DLQ, we can choose different types of DLQ [quorum, classic, stream]", type = "string")
 @ConnectorAttribute(name = "dead-letter-queue-mode", direction = INCOMING, description = "If automatically declare DLQ, we can choose different modes of DLQ [lazy, default]", type = "string")
 
+//Priority
+@ConnectorAttribute(name = "max-priority", direction = INCOMING, description = "Define priority level queue consumer", type = "int")
```

Comment:
Looking at other queue x-attributes can we rename this one to `queue.x-max-priority`?

Reply:
Hello @ozangunalp yes we can.